indicators/banistmo-g96jc7.yml

```diff
@@ -0,0 +1,30 @@
+title: Banistmo Phishing Kit G96jc7
+description: |
+    Detects a phishing kit targeting Banistmo, a bank operating in Panama and Central America. Subsidiary of Bancolombia.
+    Found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://urlscan.io/result/d46ce720-a815-4fd0-b993-341ca70029ff/
+    - https://urlscan.io/result/df1aca45-cb5d-47c2-8337-6c79b547e7f6/
+
+detection:
+
+    title:
+      html|contains:
+        - <title>Banistmo - Banca en Línea</title>
+
+    icon:
```

Comment:
I think we could improve this rule by utilizing the fact that this phishing kit uses the same GTM ID as the original page. By detecting the usage of the same GTM ID as the original page and doing a further domain check as shown below, we can accurately detect any kit using the same ID:

```yaml

gtmID:
    html|contains: "GTM-PX5MVLF"

expectedDomain:
    hostname|endswith: "banistmo.com"
```